PWGGA/GammaConv/AliCaloPhotonCuts.cxx

```diff
@@ -211,7 +211,7 @@ AliCaloPhotonCuts::AliCaloPhotonCuts(Int_t isMC, const char *name,const char *ti
   fHistNCellsBigger100MeVvsMod(NULL),
   fHistNCellsBigger1500MeVvsMod(NULL),
   fHistEnergyOfModvsMod(NULL),
-  fHistClusterEnergyvsNCells(NULL),
```

Comment:
You should have changed this line to    fHistClusterEnergyvsNCellsBeforeQA(NULL),  instead of deleting it. Please fix.